ironfish/src/network/peerNetwork.ts

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import { RollingFilter } from '@ironfish/bfilter'
```

Comment:
We can pull this out of the package.json now, and probably delete the typedefs for bfilter.d.ts too.